Packs/CommonPlaybooks/Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml

```diff
@@ -397,8 +397,13 @@ tasks:
           - operator: uniq
       email:
         complex:
-          root: Account
-          accessor: Email
+          root: Account.Email
```

Comment:
Need to make sure that changing the root accessor is not breaking compatibility for < 6.0 versions

Reply:
Would it @Itay4 ?

Reply:
probably better to keep it as it was

Reply:
Done ✔️